main_test.go

```diff
@@ -13,8 +13,10 @@ import (
 )
 
 const (
-	testAmqpURI   = "amqp://guest:guest@127.0.0.1:5672/"
-	testQueueName = "test-rabbitmq-dump-queue"
+	testAmqpURI      = "amqp://guest:guest@127.0.0.1:5872/"
```

Comment:
Why has the port changed from 5672 to 5872 ?
This breaks the automatic unit-tests in Travis CI.

Reply:
Oh dear. I changed the port locally as I had RMQ already running on port 5872.

Totally missed that one, did `go fmt` and just pushed. Sorry. 🤦‍♂